changelog/12792.txt

```diff
@@ -0,0 +1,3 @@
+```release-note: feature
```

Comment:
@hsimon-hashicorp we use a special changelog format for new features (it's outlined on the wiki). I _think_ this is more of an improvement though, so I've moved it and I'm going to tag you on an enterprise PR.